hal/network/lwip/wiznet/wiznetif_config.cpp

```diff
@@ -147,9 +147,6 @@ int WizNetifConfig::getConfigData(WizNetifConfigData* configData) {
     if (configData->reset_pin == PIN_INVALID) {
         configData->reset_pin = HAL_PLATFORM_ETHERNET_WIZNETIF_RESET_PIN_DEFAULT;
     }
-    if (configData->int_pin == PIN_INVALID) {
-        configData->int_pin = HAL_PLATFORM_ETHERNET_WIZNETIF_INT_PIN_DEFAULT;
-    }
```

Comment:
NOTE: We will have to update the docs to spell out the fact that if you want to set the INT pin back to it's default value, you must know what it is and set it explicitly.

See here: "If you want to restore one or more pins as their default value, use PIN_INVALID, and the default will be used for that pin."
https://docs.particle.io/reference/device-os/api/ethernet/pin-configuration-ethernet/

Reply:
Cc. @rickkas7